Meta/lint-ci.sh

```diff
@@ -16,6 +16,7 @@ set +e
 for cmd in \
         Meta/check-ak-test-files.sh \
         Meta/check-debug-flags.sh \
+        Meta/check-newlines-at-eof.sh \
```

Comment:
Separate script is fine as well but IMO this could just be part of check-stye.sh

Reply:
> Separate script is fine as well but IMO this could just be part of check-stye.sh

I created a separate file because `check-style.sh` only checks .cpp and .h files; also all the checks there use regexps while these new checks do not.

Reply:
> only checks .cpp and .h files

ah good point, I forgot. :+1: